app/assets/javascripts/components/students/student_list.jsx

```diff
@@ -18,7 +18,7 @@ import { notifyOverdue } from '../../actions/server_actions_redux.js';
 import CourseUtils from '../../utils/course_utils.js';
 import { getFiltered } from '../../utils/model_utils.js';
 
-const StudentList = createReactClass({
+export const StudentList = createReactClass({
```

Comment:
Since you're now exporting StudentList as a named export, I think there will be a linting error wherever the default export is being used, since it's probably doing `import StudentList`, using the same name as the named export. You can probably get around that by changing the import to `import ConnectedStudentList` or the like.

Reply:
It is not showing linting error in my vscode.

Reply:
You were right, I will make that change